loris/resolver.py

```diff
@@ -517,6 +522,21 @@ def _web_request_url(self, ident):
             options['verify'] = conf['ssl_check']
         return (url, options)
 
+    def is_resolvable(self, ident):
+        # We can raise a ``ResolverException`` in ``_web_request_url()`` if
+        # somebody tries to look up a URL that doesn't match any of the
+        # templates.  In turn, this causes ``is_resolvable()`` to rethrow
+        # that exception and return a 500 to the user.
+        #
+        # In this case, we should catch the exception and return a boolean.
+        try:
+            return super(TemplateHTTPResolver, self).is_resolvable(ident)
+        except ResolverException as exc:
+            if exc.http_status == 404:
+                return False
+            else:
+                raise
+
```

Comment:
Instead of adding this is_resolvable method, how about catching the ResolverException in SimpleHTTPResolver.is_resolvable? Wouldn't that fix the issue for both resolvers, and not require this new method?

Reply:
Ah, I hadn’t noticed that SimpleHTTPResolver can throw a ResolverException in _web_request_url() as well – yeah, I’ll move this up to the parent class.